umi dev 启动报错

## What happens?

$ umi dev

✖  error     Error: Cannot find module 'eslint-plugin-jest'
✖  error     Error: Cannot find module 'eslint-plugin-eslint-comments'
✖  error     Error: Cannot find module '@typescript-eslint/eslint-plugin'

## Mini Showcase Repository(REQUIRED)

https://github.com/xlsdg/umi-dva-antd-starter

## How To Reproduce
**Steps to reproduce the behavior:**
1. `git clone https://github.com/xlsdg/umi-dva-antd-starter.git`
2. `cd umi-dva-antd-starter && yarn && yarn start`

**Expected behavior**
1. 必须手动安装 `yarn add -D eslint-plugin-jest eslint-plugin-eslint-comments @typescript-eslint/eslint-plugin` 后才能启动工程。
2. 通过 `npm i` 安装依赖包则不会出现该问题。

## Context

- **Umi Version**: v2.8.13
- **Node Version**: v10.16.0
- **Npm Version**: v6.10.1
- **Yarn Version**: v1.17.3
- **Platform**: macOS Mojave 10.14.6

--- a/packages/umi-build-dev/src/plugins/commands/block/index.js
+++ b/packages/umi-build-dev/src/plugins/commands/block/index.js
@@ -13,8 +13,6 @@ import { getNameFromPkg } from './getBlockGenerator';
 import appendBlockToContainer from './appendBlockToContainer';
 import { gitClone, gitUpdate, getDefaultBlockList, installDependencies } from './util';
 import clearGitCache from './clearGitCache';
-import tsToJs from './tsTojs';
-import removeLocale from './remove-locale';
 
 export default api => {
   const { log, paths, debug, applyPlugins, config } = api;
@@ -216,13 +214,13 @@ export default api => {
     // 调用 sylvanas 转化 ts
     if (js) {
       spinner.start('🤔  TypeScript to JavaScript');
-      tsToJs(generator.blockFolderPath);
+      require('./tsTojs').default(generator.blockFolderPath);
       spinner.succeed();
     }
 
     if (uni18n) {
       spinner.start('🌎  remove i18n code');
-      removeLocale(generator.blockFolderPath, uni18n);
+      require('./remove-locale').default(generator.blockFolderPath, uni18n);
       spinner.succeed();
     }
 

